test/synapse_test.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 git clone https://github.com/vertexproject/synapse.git  ./synapse
 cd ./synapse
-pytest -v -s -n auto --disable-warnings --durations 6 --maxfail 6 --cov synapse --no-cov-on-fail --cov-report=term
```

Comment:
Running tests in parallel was breaking the Axon tests. Although I don't think it should cause them to fail, it was, so I'm removing it for now.